commands/jobs.go

```diff
@@ -30,7 +30,7 @@ func (command *JobsCommand) Execute([]string) error {
 	var jobs []atc.Job
 
 	jobs, err = target.Team().ListJobs(pipelineName)
-	headers = []string{"name", "paused", "status"}
+	headers = []string{"name", "paused", "status", "current"}
```

Comment:
I think "current" is a bit confusing; that could also mean the current finished build. Maybe "next" or something? "Running"? Not sure.

Reply:
changed in commit 87228af

Reply:
(specifically, I changed it to "next" since that's already the term the code uses for that build)